golem-component-service/src/api/component.rs

```diff
@@ -421,6 +422,29 @@ impl ComponentApi {
         record.result(response)
     }
 
+    #[oai(path = "/search", method = "post", operation_id = "search_components")]
+    async fn get_components_batch_post(
+        &self,
+        components_search: Json<Vec<ComponentSearchParameters>>,
```

Comment:
I would suggest not using a raw array as the top level json type. That will make it very difficult to change this endpoint in a backward compatible way (e.g. adding pagination).
Better would be to have an object as the top level (even when it only has a single array field for now)

Reply:
Makes sense

Reply:
Resovled